Installation error: implicit instantiation of undefined template

## Issue summary
When running the installation instructions, I encountered the error `error: implicit instantiation of undefined template 'std::basic_stringstream<char>' std::stringstream s;`. 

## Proposed solution:
This was resolved by adding `#include <sstream>` to `src/operation.h`. 
I'd be happy to open a PR to add this, but I'm not sure what's the preferred contribution guideline.

## Additional context:
Full trace in the terminal:
```
(base) ➜  cmake-build-release git:(master) make
[  2%] Building CXX object CMakeFiles/exactextract_gdal.dir/src/deferred_gdal_writer.cpp.o
In file included from ~/Documents/GitHub/exactextract/src/deferred_gdal_writer.cpp:15:
~/Documents/GitHub/exactextract/src/operation.h:131:39: error: implicit instantiation of undefined template 'std::basic_stringstream<char>'
                    std::stringstream s;
                                      ^
/Library/Developer/CommandLineTools/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/iosfwd:146:32: note: template is declared here
    class _LIBCPP_TEMPLATE_VIS basic_stringstream;
                               ^
In file included from ~/Documents/GitHub/exactextract/src/deferred_gdal_writer.cpp:15:
~/Documents/GitHub/exactextract/src/operation.h:138:39: error: implicit instantiation of undefined template 'std::basic_stringstream<char>'
                    std::stringstream s;
                                      ^
/Library/Developer/CommandLineTools/SDKs/MacOSX13.3.sdk/usr/include/c++/v1/iosfwd:146:32: note: template is declared here
    class _LIBCPP_TEMPLATE_VIS basic_stringstream;
                               ^
2 errors generated.
make[2]: *** [CMakeFiles/exactextract_gdal.dir/src/deferred_gdal_writer.cpp.o] Error 1
make[1]: *** [CMakeFiles/exactextract_gdal.dir/all] Error 2
make: *** [all] Error 2
```
System configurations:
- MacOS: 13.6.1 (22G313)
```
(base) ➜  gcc --version
Apple clang version 15.0.0 (clang-1500.0.40.1)
Target: arm64-apple-darwin22.6.0
Thread model: posix
InstalledDir: /Library/Developer/CommandLineTools/usr/bin
(base) ➜ cmake --version
cmake version 3.27.7

CMake suite maintained and supported by Kitware (kitware.com/cmake).
(base) ➜ gdalinfo --version
GDAL 3.7.3, released 2023/10/30
(base) ➜ geos-config --version
3.12.0
```

--- a/src/operation.h
+++ b/src/operation.h
@@ -16,6 +16,7 @@
 
 #include <functional>
 #include <string>
+#include <sstream>
 
 #include "feature.h"
 #include "grid.h"

